draft-ietf-rats-corim.md

```diff
@@ -1259,6 +1259,22 @@ for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's appraisal policy, which is often driven by the complexity and nature of
+use case.
+
+In a certain use case CoBOM's may not be required at all. For example, where only limited CoRIMs
```

Comment:
This is inconsistent with the requirement in L1257-1258.

Reply:
Good catch, we need to word-smith L1257-1258 to state that when a Verifier policy requires CoBOM then,  Data contained in a tag MUST NOT be used
for appraisal until a CoBOM which activates that tag has been received and successfully processed.

Reply:
I will make the change!

Reply:
Ok, I have done in the new revision!